src/buy-sell.js

```diff
@@ -3,21 +3,11 @@ module.exports = BuySell;
 
 // var buySell = new Blockchain.BuySell(Blockchain.MyWallet.wallet);
 function BuySell (wallet, debug) {
-  this.debug = Boolean(debug);
-
-  /* istanbul ignore if */
-  if (this.debug) {
-    console.info('BuySell debug enabled');
-  }
-
   this._wallet = wallet;
 
   // Stop if 2nd password is enabled
   if (wallet.external === null) {
-    /* istanbul ignore if */
-    if (this.debug) {
-      console.info("2nd password enabled, don't initialize BuySell");
-    }
+    console.info("2nd password enabled, don't initialize BuySell");
```

Comment:
Actually that's not necessary (according to the linter), because of the single quote in `don't`.